src/js/common/messageHelper.js

```diff
@@ -12,19 +12,26 @@ var gpii   = fluid.registerNamespace("gpii");
 
 fluid.registerNamespace("gpii.handlebars.helper.messageHelper");
 
-gpii.handlebars.helper.messageHelper.getHelper = function () {
+gpii.handlebars.helper.messageHelper.getHelper = function (that, serverAware) {
     return function (messageKey, dataOrRootContext, rootContext) {
         if (arguments.length < 2) {
             fluid.fail("You must call the 'messageHelper' helper with at least a message key.");
         }
         else {
-            // Pick up the message bundles from the root context, which is always the last argument.
-            var messages = fluid.get(rootContext || dataOrRootContext, "data.root.messages");
-
+            if (serverAware) {
```

Comment:
This block (and the next) have multiple linting problems:

```
/Users/duhrer/Source/rtf/gpii-handlebars.sgithens/src/js/common/messageHelper.js
  22:21  error  'messages' used outside of binding context  block-scoped-var
  27:21  error  'messages' is already defined               no-redeclare
  27:21  error  'messages' used outside of binding context  block-scoped-var
  30:17  error  'data' is defined but never used            no-unused-vars
  31:65  error  'messages' used outside of binding context  block-scoped-var
  31:65  error  'messages' used outside of binding context  block-scoped-var
```

Reply:
cleaned this up, although there are obviously still issues to look at in the block